navigation/navbar.tsx

```diff
@@ -25,7 +25,18 @@ export default function NavBar() {
     <Tab.Navigator
       initialRouteName={mainName} // ask about implementing login as initial page display
       screenOptions={({ route }) => ({
-        tabBarIcon: ({ focused, color, size }) => {
+        tabBarShowLabel: false,
+        tabBarStyle: {
+          backgroundColor: "#328281",
+        },
+        tabBarActiveBackgroundColor: "#408E88",
+        tabBarItemStyle: {
+          borderRadius: 15,
+        },
```

Comment:
could you add a touch of margin to this? the highlight intersects with the curved edge of the phone screen and it's giving me the heebie-jeebies